library/utils.py

```diff
@@ -37,9 +37,6 @@ def format_url(path: str, subpath: str) -> str:
     path = path[:-1] if path[-1] == "/" else path
     url = path if len(subpath) == 0 else path + "/" + subpath
 
-    if "http" in url and ".zip" not in url:
```

Comment:
Do we not need /vsicurl/ for non-socrata URLs that begin with http?

Reply:
no we don't